### ISSUE TITLE
Add Keyboard to Lichess natively?

### ISSUE THREAD

**Issue by mr-yt12** [mr-yt12/lichess-gui-use-code-example#37]
https://github.com/ornicar/chessground

Research about the possibility to implement KB / multipremove directly into Lichess. However, chessground code is already quite complex, so it should perhaps be in a separate, easily manageable component. I guess just KB could be a good start, and multipremoves could be added later? Or vice versa? If the player has it enabled, it only activates when playing against another player who has it enabled. It should have its own option in the Game Behavior settings. It should not be possible to change during a game, the changes will only be applied to the next games (to avoid people disabling it before the game and enabling it during the game to prevent the opponent from using it).

**Comment by randomguyreborn** [comment]
@mr-yt12 Yeah its possible to add kb to lichess natively but the reason why they didn't allow kb is **Extensions that assist with playing moves have denatured fast chess to the point that many players won't play it anymore. We believe that banning keyboard extensions will result in UltraBullet and HyperBullet becoming more popular** so not sure whether they accept our suggestions,  also the sentence **it only activates when playing against another player who has it enabled**. okay this is fine but only applies to pc and laptop players so what does phone or tab players do against their opponents?

**Comment by randomguyreborn** [comment]
@mr-yt12 Yeah its possible to add kb to lichess natively but they didn't allow kb so not sure whether they accept our suggestions,  also the sentence **it only activates when playing against another player who has it enabled**. okay this is fine but only applies to pc and laptop players so what does phone or tab players do against their opponents?
